File: Program20.cpp
#include<iostream>
using namespace std;

class Number
{
   public:

     void Display()
     {
       int iCnt = 0;
    
       for(iCnt = 1; iCnt<=5; iCnt++)
       {
          cout<<iCnt<<endl;
       }
     }
};

int main()
{
    Number nobj;
    nobj.Display();

    return 0;
}

File: Program20.java
import java.lang.*;

class Number 
{   

    public void Display()
    {
        int iCnt = 0;
        
        for(iCnt = 1; iCnt<=5; iCnt++)
        {
            System.out.println(iCnt);
        }
    }
}
class Program20
{
    public static void main(String arg[])
    {
        Number nobj = new Number();

        nobj.Display();

    }
}
